wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/OptionElement.java

```diff
@@ -36,99 +34,82 @@
     OPTION
   }
 
-  public static OptionElement create(String name, Kind kind, Object value) {
-    return create(name, kind, value, false);
-  }
-
-  public static OptionElement create(String name, Kind kind, Object value,
-      boolean isParenthesized) {
-    return new AutoValue_OptionElement(name, kind, value, isParenthesized);
-  }
-
-  public abstract String name();
-  public abstract Kind kind();
-  public abstract Object value();
-  public abstract boolean isParenthesized();
-
-  public final String toSchema() {
-    Object value = value();
-    switch (kind()) {
-      case STRING:
-        return formatName() + " = \"" + value + '"';
-      case BOOLEAN:
-      case NUMBER:
-      case ENUM:
-        return formatName() + " = " + value;
-      case OPTION: {
-        StringBuilder builder = new StringBuilder();
-        OptionElement optionValue = (OptionElement) value;
+  fun toSchema(): String = buildString {
+    when (kind) {
+      Kind.STRING -> return """${formatName()} = "$value""""
```

Comment:
In this case we end up throwing away the `StringBuilder` instance, maybe call `buildString` only for cases where it's needed?

Reply:
changed to `append`